lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -265,24 +270,41 @@ private OkHttpAetherClient newHttpClient() {
 				proxyPort = Integer.getInteger(System.getProperty("https.proxyPort"));
 			}
 		}
-		if (proxyHost != null && proxyPort != null) {
-			AetherClientProxy clientProxy = new AetherClientProxy();
-			clientProxy.setHost(proxyHost);
-			clientProxy.setPort(proxyPort);
-			clientProxy.setAuthentication(authentication);
-			config.setProxy(clientProxy);
-		}
-
-		if (config.getProxy() == null) {
-			config.setAuthentication(authentication);
-		}
-		
+//		if (proxyHost != null && proxyPort != null) {
+//			HttpProxy clientProxy = new AetherClientProxy();
+//			clientProxy.setHost(proxyHost);
+//			clientProxy.setPort(proxyPort);
+//			clientProxy.setAuthentication(authentication);
+//			
+//			config.setProxy(clientProxy);
+//		}
```

Comment:
Let's just remove if this isn't necessary any more.

Reply:
Removed